### ISSUE TITLE
Ideas for the wiki

### ISSUE THREAD

**Issue by elmar-hinz** [keatontaylor/alexa-actions#135]
I plan to rework the Wiki. That's what I consider to do. What do you think?

### Installation

Document the new import feature as the standard way to install.  Move the classical way by copying single files to a different page, while keeping it available as a fallback solution.

### Hello world

The page *Setting up your first actionable notification* is describing a door. Not everybody is in possession of a smart lock. I would like to move this to the examples section and do something more simple, basically a question for yes or no.

The yes-no question is also suitable to describe it as a simple *node red* flow, to offer an alternative starting point.

**Comment by DEADSEC-SECURITY** [comment]
Yup seems like a good idea. If you need help let me know. Do you have discord or something like that. Probably easier if you need to ask me anything.

**Comment by elmar-hinz** [comment]
![Screenshot 2022-12-29 at 10 10 50](https://user-images.githubusercontent.com/1227339/209985626-46e22568-d048-4a63-b3f0-7bda80afddb9.png)

This is hello world done in red node, to display the idea. With traditional automation I would replace trigger and debugger with a virtual button and light.

**Comment by DEADSEC-SECURITY** [comment]
@elmar-hinz are you authorized to vhange the wiki? 
Can you send a copy of the template code?

**Comment by DEADSEC-SECURITY** [comment]
We should also include a section for people contributing for new languages explaining what files need to be modded

**Comment by DEADSEC-SECURITY** [comment]
Updated the wiki to reflect this. Just the home page for now https://github.com/keatontaylor/alexa-actions/wiki
![image](https://user-images.githubusercontent.com/45077772/209997125-82b1a2f8-1c00-44ad-8b02-f1c216f89c3e.png)

**Comment by DEADSEC-SECURITY** [comment]
@elmar-hinz can you confirm if you modded the wiki? I dont want to be overwriting your stuff

**Comment by elmar-hinz** [comment]
Yes, I can edit the wiki. I guess everybody is allowed to do.

**Comment by elmar-hinz** [comment]
@DEADSEC-SECURITY I am currently not working the wiki. I am doing the supper.

**Comment by elmar-hinz** [comment]
Current state

TODO

- Hello World Node Red
- Personal language translation
- Language contribution to GitHub

DONE:

- Import
- Hello World Native

**Comment by elmar-hinz** [comment]
About troubleshooting pages

I tried to avoid a lot of troubles upfront by giving prerequisites, screenshots and showing how to do intermediate success checking.

With improved import and documentation more ppl will use the skill. That in turn may bring on more issue tickets. To really know, where troubles pop up, we just have to see.

The issue with the wife's account was quite interesting. On the first glance, it looks like a funny human assumption, that the devices within the same room are reachable, even if they belong to different Amazon accounts.

Having a deeper look, all devices are connected to Home Assistant. Home Assistant doesn't know about different Amazon accounts. All devices within the home are connected to it. Home Assistant actually suggests the wife's devices for selection. So this issue may occur more often, than you would initially anticipate.

**Comment by DEADSEC-SECURITY** [comment]
New issue report is coming which should help us with this.

**Comment by DEADSEC-SECURITY** [comment]
https://github.com/keatontaylor/alexa-actions/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml new issue report done. Let me know if you think we should add anything relevant

**Comment by elmar-hinz** [comment]
I close this ticket for now. The original concerns of ***Hello World*** and ***Installation*** are fully done by ***Getting Started*** of the Wiki meanwhile. 

The issue template is a different topic.
